includes/admin/js/geolocator.js

```diff
@@ -1,53 +1,194 @@
 ( function( $ ) {
-	var accessToken = $('#wanderlist-geolocation-input').data('mapboxkey');
-	$( '#wanderlist-geolocation-input' ).on('change', function() {
-		var locationString = $( this ).attr( 'value' );
-		var geocodeURI = 'https://api.mapbox.com/v5/geocode/mapbox.places/' + encodeURI( locationString ) + '.json?access_token=' + accessToken;
-		$.ajax(geocodeURI, {
-			success: function(response) {
-				var result = response.features[0];
-				var selected = result.context;
-				// If the context object doesn't exist, we're dealing with a limited dataset. Let's get what we can.
-				if (!selected) {
-					if ( result.id.match( /^country/ ) ) {
-						var country = result.text;
-					} else if ( result.id.match( /^region/ ) ) {
-						var region = result.text;
-					}
-				// We have a full result set, so let's parse its data
-				} else {
-					var city = result.text;
-					for (var i = 0; i < selected.length; i++) {
-						if ( selected[i].id.match( /^country/ ) ) {
-							var country = selected[i].text;
-						} else if ( selected[i].id.match( /^region/ ) ) {
-							var region = selected[i].text;
-						}
-					}
+	// Grab our access token (we need an error if it isn't set!)
+	if ( ! $( '#wanderlist-geolocation-input' ).data( 'mapboxkey' ) ) {
+		console.log( 'No key set.' );
+
+	} else {
+
+		// Initialise map variables and settings
+		L.mapbox.accessToken = $( '#wanderlist-geolocation-input' ).data( 'mapboxkey' );
+		var map = L.mapbox.map( 'wanderlist-geolocation-map', 'mapbox.streets', {
+			zoomControl: false,
+			attributionControl: false
+		} );
+
+		// Disable zoom
+		map.scrollWheelZoom.disable();
+
+		// Re-add a zoom control at bottom right
```

Comment:
Re-add is because it's removed above with `zoomControl: false,`
Basically this is how I can change the position of this control, which otherwise goes at the top left.
I could leave it there but I'd need to move the data entry field elsewhere, and I *think* top left is the most sensible place—although I'm not entirely sure of that one yet.